detect/Candidate[] candidates
# detect/RotatedBoundingBox bbox
detect/PointTuple2D center
int32 target_index